addon/ember-test-waiters/index.js

```diff
@@ -0,0 +1 @@
+export * from '@ember/test-waiters';
```

Comment:
We should add an issue to add a deprecation here I think.

Reply:
Agree.